src/Nethermind/Nethermind.Serialization.Rlp/NettyRlpStream.cs

```diff
@@ -102,5 +102,11 @@ public override int Position
         public override bool HasBeenRead => _buffer.ReadableBytes > 0;
 
         protected override string Description => "|NettyRlpStream|description missing|";
+        public Span<byte> AsSpan => _buffer.Array.AsSpan().Slice(_buffer.ArrayOffset + _initialPosition, _buffer.WriterIndex - _initialPosition);
```

Comment:
Can we have `AsSpan` extension method to buffer?
Not sure if this should be property or method.

Reply:
You mean change it to method?

Reply:
Yes extract it!

Reply:
Done